contracts/Common/common.hpp

```diff
@@ -18,6 +18,8 @@ using namespace eosio;
 #define EOS_SYMBOL symbol("EOS", EOS_PRECISION)
 #define MAX_AMOUNT asset::max_amount
 #define MAX_RATE 1000000 /* up to 1M tokens per EOS */
+#define STAKE_ACCOUNT "eosio.stake"_n
```

Comment:
just to make sure this covers all stakes? cpu, bandwith and voting?

Reply:
yes.